airbyte-integrations/connectors/source-instatus/source_instatus/instatus.yaml

```diff
@@ -0,0 +1,286 @@
+      - stream: "*ref(definitions.maintenances_stream)"
+        parent_key: "updates_ids"
+        stream_slice_field: "maintenance_update_id"
+
+  maintenance_updates_stream:
+    schema_loader:
+      $ref: "*ref(definitions.schema_loader)"
+    $options:
+      name: "maintenance_updates"
+      primary_key: "id"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.api_error_handler_requester)"
+        path: "v1/{{ stream_slice.parent_slice.page_id }}/maintenances/{{ stream_slice.updates_object_id }}/maintenance-updates/{{ stream_slice.maintenance_update_id }}"
+      stream_slicer:
+        $ref: "*ref(definitions.maintenance_updates_stream_slicer)"
+
+  templates_stream:
+    schema_loader:
+      $ref: "*ref(definitions.schema_loader)"
+    $options:
+      name: "templates"
+      primary_key: "id"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.api_error_handler_requester)"
+        path: "v1/{{ stream_slice.page_id }}/templates"
+      stream_slicer:
+        $ref: "*ref(definitions.page_stream_slicer)"
+
+  team_stream:
+    schema_loader:
+      $ref: "*ref(definitions.schema_loader)"
+    $options:
+      name: "team"
+      primary_key: "id"
+    retriever:
+      $ref: "*ref(definitions.user_field_retriever)"
+      requester:
+        $ref: "*ref(definitions.api_error_handler_requester)"
+        path: "v1/{{ stream_slice.page_id }}/team"
+      stream_slicer:
+        $ref: "*ref(definitions.page_stream_slicer)"
+
+  subscribers_stream:
+    schema_loader:
+      $ref: "*ref(definitions.schema_loader)"
+    $options:
+      name: "subscribers"
+      primary_key: "id"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.api_error_handler_requester)"
+        path: "v1/{{ stream_slice.page_id }}/subscribers"
+      stream_slicer:
+        $ref: "*ref(definitions.page_stream_slicer)"
+
+  metrics_stream:
+    schema_loader:
+      $ref: "*ref(definitions.schema_loader)"
+    $options:
+      name: "metrics"
+      primary_key: "id"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.api_error_handler_requester)"
+        path: "v1/{{ stream_slice.page_id }}/metrics"
+      stream_slicer:
+        $ref: "*ref(definitions.page_stream_slicer)"
+
+  page_url_stream_slicer:
+    type: SubstreamSlicer
+    parent_stream_configs:
+      - stream: "*ref(definitions.status_pages_stream)"
+        parent_key: "subdomain"
+        stream_slice_field: "url"
+
+  public_data_stream:
+    schema_loader:
+      $ref: "*ref(definitions.schema_loader)"
+    $options:
+      name: "public_data"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.public_data_requester)"
+        path: "https://{{ stream_slice.url }}.instatus.com/summary.json"
```

Comment:
To stick with convention we should separate the `base_url` and the `path`:
`url_base: "https://{{ stream_slice.url }}.instatus.com/"`
`path: "summary.json"`

Reply:
done

Reply:
I returned the previous version because it doesn't have `stream_slice` at the url_base step, so I received an error while SAT.

Reply:
ah i see. sorry for the mistake